news/2017-07-22-beatmap-spotlight-june-2017.md

```diff
@@ -0,0 +1,106 @@
+
++ **osu!:** [Toy](https://osu.ppy.sh/u/2757689)
++ **osu!taiko:** [Quizmaegi](https://osu.ppy.sh/u/2228401)
++ **osu!catch:** [_Asriel](https://osu.ppy.sh/u/566276)
++ **osu!mania:** [Xcstasy](https://osu.ppy.sh/u/1306875)
+
+
+## osu!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/youkai-no-yama.jpg)](https://osu.ppy.sh/s/458983)
+
+Absolute mastery of a style at its finest. **[GoldenWolf](https://osu.ppy.sh/u/1612624)**'s latest marathon takes us on a journey through this epic piece by Demetori. The structure of the map is simple, yet refined. The gameplay experience is difficult, yet accompanies the song perfectly. And not to mention the classic meticulous GoldenWolf hitsounding! A well-deserved top placing on this month's charts.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/mahou-shoujo-chino.jpg)](https://osu.ppy.sh/s/458983)
+
+After No Poi, **[Asahina Momoko](https://osu.ppy.sh/u/3650145)** returns again in 2017 to bring us more Chino - and good mapping. Asahina Momoko never sticks to the 'mapping-meta', and as such, all his maps are always different and interesting, With his particular style, he's managed to capture this song perfectly with strong innovative spacing usage to create different intensities. His rhythm usage is top notch, being consistent enough to maintain playability, while having ample variation so it does not bore out the player with its length. You won't regret giving this one a good play through!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/dorchadas.jpg)](https://osu.ppy.sh/s/580215)
+
+**[Delis](https://osu.ppy.sh/u/1603923)** continues to impress, taking yet another spot in this month's spotlights. The map perfectly represents the intensity changes in the song. Combined with great flow and variety this makes an overall great experience. Definitely a map to check out!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/hyadain.jpg)](https://osu.ppy.sh/s/554297)
+
+This month, **[Okorin](https://osu.ppy.sh/u/1623405)** brings us a more modern re-map of a song that has already been ranked multiple times before. The higher difficulties captivate with fun jumps and well-placed symmetry usage. The lower difficulties aren't of less quality though, since they manage to mirror the interesting rhythm and gameplay elements from the higher difficulties while still being easy enough. This beautiful spread is worth checking out for players of every skill level!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/innocence.jpg)](https://osu.ppy.sh/s/557231)
+
+Calm, yet very intense - this describes both song and mapset. **[appleeaterx](https://osu.ppy.sh/u/2407160)** has fulfilled a great job in emphasizing the right spots, to give the player the best feeling possible to fully enjoy this song. Definitely give it a try!
```

Comment:
I changed the second sentence a little to might make it clearer but if you want further changes you have to speak with Irreversible as I won't re-write the entire comment, and he doesn't seem to understand what you expect from him here.